fe/fe-core/src/main/java/org/apache/doris/catalog/DataProperty.java

```diff
@@ -46,6 +46,8 @@ public class DataProperty implements Writable {
     private String remoteStorageResourceName;
```

Comment:
If Migration Handler is in BE, remoteStorageResourceName and remoteCooldownTimeMs are not needed, just remove them. The serialization for this class is a json.